### PR TITLE
docs: nav updates for user, role, team management

### PR DESCRIPTION
**Pull request by imatwawana**
Updates navigation-related text and images in Administration > Roles and Permissions, Team, and User management.

### CODE REVIEW

**Review by Eve832 — APPROVED** [review]

File: docs/sources/administration/data-source-management/index.md [hunk 1]
```diff
@@ -25,22 +25,18 @@ Before you can create your first dashboard, you need to add your data source.
 
 **To add a data source:**
 
-1. Select the cog icon on the side menu to show the configuration options.
+1. Click **Administration** in the left-side menu.
 1. Select **Data sources**.
 
    This opens the data sources page, which displays a list of previously configured data sources for the Grafana instance.
```
Comment (imatwawana) [inline docs/sources/administration/data-source-management/index.md:31]:
```suggestion
1. Click **Connections** in the left-side menu.
```

File: docs/sources/administration/data-source-management/index.md [hunk 2]
```diff
@@ -25,22 +25,18 @@ Before you can create your first dashboard, you need to add your data source.
...
 
-1. Select **Add data source** to see a list of all supported data sources.
-
-   {{< figure src="/static/img/docs/v75/add-data-source-7-5.png" max-width="600px" class="docs-image--no-shadow">}}
+1. Select **+ Add new data source** to see a list of all supported data sources.
 
 1. Enter the name of a specific data source in the search dialog.
 
    You can also scroll through supported data sources grouped into time series, logging, tracing, and other categories.
```
Comment (imatwawana) [inline docs/sources/administration/data-source-management/index.md:37]:
```suggestion
1. Enter the name of a specific data source in the search dialog.
```

Comment (imatwawana) [inline docs/sources/administration/data-source-management/index.md:37]:
```suggestion
1. Enter the name of a specific data source in the search dialog.

   You can also filter by **Data source** first.
```

File: docs/sources/administration/data-source-management/index.md [hunk 3]
```diff
@@ -25,25 +25,9 @@ Before you can create your first dashboard, you need to add your data source.
...
-
-1. Move the cursor over the data source you want to add.
-
-1. Click **Select**.
-
-   This opens the data source configuration page.
-
+1. Click **Connections** in the left-side menu.
+1. Enter the name of a specific data source in the search dialog. You can filter by **Data source** to only see data sources.
+1. Click the data source you want to add to open the configuration page.
```
Comment (imatwawana) [inline docs/sources/administration/data-source-management/index.md:30]:
```suggestion
1. Click the data source you want to add.
```

File: docs/sources/administration/data-source-management/index.md [hunk 4]
```diff
@@ -25,80 +25,52 @@ Before you can create your first dashboard, you need to add your data source.
...
+You can configure data source permissions to allow or deny certain users the ability to query or edit a data source. Each data source’s configuration includes a Permissions tab where you can restrict data source permissions to specific users, teams, or roles.
 
 > **Note:** Available in [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise/" >}}) and [Grafana Cloud Pro and Advanced](/docs/grafana-cloud).
 
-### Enable data source permissions
-
-By default, data sources in an organization can be queried by any user in that organization. For example, a user with the `Viewer` role can issue any possible query to a data source, not just
-queries that exist on dashboards they have access to.
-
-When permissions are enabled for a data source in an organization, the user who created the datasource can edit the datasource and in addition, viewers can query the datasource.
```
Comment (IevaVasiljeva) [inline docs/sources/administration/data-source-management/index.md:63]: Might be worth to keep some of this in. Maybe something like:

`By default, data sources in an organization can be queried by any user in that organization and edited by the user who created the data source and users with `Admin` role. For example, a user with the `Viewer` role can issue any possible query to a data source, not just queries that exist on dashboards they have access to.`

**Review by IevaVasiljeva — COMMENTED** [review]
Thanks for updating the docs related to data source permissions! I left a small suggestion, but otherwise it looks great to me.

File: docs/sources/administration/data-source-management/index.md (hunk 4 above)
Comment (imatwawana) [reply docs/sources/administration/data-source-management/index.md:63]: It's actually still there but further down. But I'll add in the bit you've got about Admin roles as I think that's good too!

File: docs/sources/administration/data-source-management/index.md [hunk 5]
```diff
@@ -25,80 +25,52 @@ Before you can create your first dashboard, you need to add your data source.
...
+1. On the Permissions tab, click **Add a permission**.
+1. Select **User**, **Team**, or **Role**.
+1. Select the entity for which you want to modify permissions.
 1. Select the **Query** or **Edit** permission.
 1. Click **Save**.
 
 <div class="clearfix"></div>
 
-### Disable data source permissions
+### Edit user, team, or role permissions
```
Comment (imatwawana) [inline docs/sources/administration/data-source-management/index.md:59]:
```suggestion
### Edit data source permissions for users, teams, or roles
```

File: docs/sources/administration/data-source-management/index.md [hunk 6]
```diff
@@ -25,80 +25,52 @@ Before you can create your first dashboard, you need to add your data source.
...
+1. Navigate to **Administration > Data sources**.
+1. Select the data source for which you want to edit permissions.
+1. On the Permissions tab, find the user, team, or role permission you want to update.
+1. Select a different option in the **Permission** dropdown.
 
-Note that _all_ existing permissions created for the data source will be deleted.
+<div class="clearfix"></div>
 
-**Disable permissions for a data source:**
+### Remove user, team, or role permissions
```
Comment (imatwawana) [inline docs/sources/administration/data-source-management/index.md:68]:
```suggestion
### Remove data source permissions for users, teams, or roles
```

File: docs/sources/administration/data-source-management/index.md [hunk 7]
```diff
@@ -25,80 +25,52 @@ Before you can create your first dashboard, you need to add your data source.
...
-queries that exist on dashboards they have access to.
-
-When permissions are enabled for a data source in an organization, the user who created the datasource can edit the datasource and in addition, viewers can query the datasource.
-
-**Enable permissions for a data source:**
-
-1. Navigate to **Configuration > Data Sources**.
-1. Select the data source you want to enable permissions for.
-1. On the Permissions tab, click **Enable**.
+By default, data sources in an organization can be queried by any user in that organization. For example, a user with the `Viewer` role can issue any possible query to a data source, not just queries that exist on dashboards they have access to.
```
Comment (imatwawana) [inline docs/sources/administration/data-source-management/index.md:41]:
```suggestion
By default, data sources in an organization can be queried by any user in that organization. For example, a user with the `Viewer` role can issue any possible query to a data source, not just queries that exist on dashboards to which they have access. Additionally, by default, data sources can be edited by the user who created the data source, as well as users with the `Admin` role. 
```

**Review by Eve832 — APPROVED** [review]
lgtm!

**Pull request merged**